packages/fxa-payments-server/public/locales/en-US/main.ftl

```diff
@@ -66,7 +85,27 @@ payment-cvc =
   .label = CVC
 payment-zip =
   .label = ZIP code
-payment-confirm = I authorize Mozilla, maker of Firefox products, to charge my payment method <strong>${ $amount } per { $interval }</strong>, according to payment terms, until I cancel my subscription.
+  
+payment-confirm-day = { $intervalCount -> 
+  [one] I authorize Mozilla, maker of Firefox products, to charge my payment method <strong>${$amount} daily</strong>, according to payment terms, until I cancel my subscription.
```

Comment:
Mozilla and Firefox are brand names, should be treated as variables: 
{ -brand-name-mozilla } 
{ -brand-name-firefox }

If these are the only brands used in the overall FxA files, you can define them at the beginning of the file. If there are more to come, you can have a separate brands.ftl  that other files can reference to.

Reply:
Each of these updates were made in separate branches as to not block one another, so I did not see that the formats ended up identical for those translations. As to why they are separate: It is possible that at one point in time they were not the same.

Given they are in two different flows, I would like to get @johngruen 's input on if we can merge them.